client/libs/core-state/src/lib/media/media.actions.ts

```diff
@@ -0,0 +1,49 @@
+import { createAction, props } from '@ngrx/store';
+
+import { Media } from '@sb/core-data';
+
+export const selectedMedia = createAction(
+  '[Media] Media Selected',
+  props<{ selectedMediaId: string }>()
+);
+
+// Load Actions
+export const loadMedia = createAction('[Media] Load Media');
+
+export const loadMediaSuccess = createAction(
+  '[Media] Load Media Success',
+  props<{ media: Media[] }>()
+);
+
+// Create Actions
+export const createMedia = createAction(
+  '[Media] Create New Media',
+  props<{ media: Media }>()
+);
+
+export const createMediaSuccess = createAction(
+  '[Media] New Media Created Successfully',
```

Comment:
```suggestion
  '[Media] Media Successfully Created',
```

Reply:
Make this change on all the other feature actions as well, sermons, speakers etc.